frontend/src/app/selectors/experiment.js

```diff
@@ -0,0 +1,30 @@
+import { createSelector } from 'reselect';
```

Comment:
Why separate these from the reducer? Thought the [standard advice](https://twitter.com/dan_abramov/status/664581975764766721) was to export selectors from the reducer module.